web/package.json

```
@@ -63,25 +66,27 @@
     "msw-storybook-addon": "^2.0.5",
     "prettier": "^3.6.2",
     "storybook": "^9.1.10",
+    "typescript": "^5.9.3",
     "vite": "^7.1.11",
     "vitest": "^3.2.4"
   },
   "scripts": {
     "build": "vite build",
     "check": "npm run check:eslint && npm run check:prettier",
```

以下のように修正するとtypescriptでのエラーが出力されるようになりました
```suggestion
    "check": "npm run check:eslint && npm run check:prettier && npm run check:tsc",
    "check:eslint": "eslint --max-warnings=0 './src/**/*.{js,jsx,ts,tsx}'",
    "check:prettier": "prettier --check './src/**/*.{js,jsx,ts,tsx}'",
    "check:tsc": "tsc --noEmit",
```